example_project/core/migrations/0001_initial.py

```diff
@@ -1,5 +1,4 @@
 # -*- coding: utf-8 -*-
```

Comment:
You can remove these encoding cookies. Python 3 defaults to reading source files as utf-8.

Reply:
Normally I would and in this case I simply forgot.

About a month ago it was brought to my attention that this might be unrelated to Python 2/3 why I wonder if there are any downsides to removing them.  I started with Python 3 and never really used Python 2.  If anyone could enlighten me, I would really appreciate it. :)
ref https://github.com/celery/celery/pull/5288#pullrequestreview-221080631 and https://www.python.org/dev/peps/pep-0263/